haiku-apps/batchrename/batchrename-0.2.0.recipe

```diff
@@ -0,0 +1,48 @@
+SUMMARY="An application and Tracker add-on to rename multiple files at once"
+DESCRIPTION="batchrename is a standalone application, and Tracker add-on for Haiku. \
+You can rename multiple files in multiple directories at once, and also shows \
+a preview. It offers many renaming options, and can also rename files by contents \
+of their attributes, or the result of shell scripts in case you need that."
+HOMEPAGE="https://github.com/axeld/rename"
+COPYRIGHT="2019-2024 pinc Software"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/axeld/rename/archive/39a87c5d5127544f831e89fce91c2b33193cc103.tar.gz"
+CHECKSUM_SHA256="085bd2ceb244b6bd44e19903fe5efc08567bb05d42aa0e1dd88f554b756ec58a"
+SOURCE_DIR="rename-39a87c5d5127544f831e89fce91c2b33193cc103"
```

Comment:
Suggestion, as this is a commit checkout we mostly use the variable srcGitRev to define the tag and rename the recipe to reflect that it's a git checkout: eg `batchrename-0.2.0~git.recipe`
```diff
-SOURCE_URI="https://github.com/axeld/rename/archive/39a87c5d5127544f831e89fce91c2b33193cc103.tar.gz"
-CHECKSUM_SHA256="085bd2ceb244b6bd44e19903fe5efc08567bb05d42aa0e1dd88f554b756ec58a"
-SOURCE_DIR="rename-39a87c5d5127544f831e89fce91c2b33193cc103"
+srcGitRev="39a87c5d5127544f831e89fce91c2b33193cc103"
+SOURCE_URI="https://github.com/axeld/rename/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="085bd2ceb244b6bd44e19903fe5efc08567bb05d42aa0e1dd88f554b756ec58a"
+SOURCE_FILENAME="batchrename-$portVersion-$srcGitRev.tar.gz"
+SOURCE_DIR="rename-$srcGitRev"
```

Reply:
On a future update only the srcGitRev needs to be updated then.

Reply:
Thanks for testing, and the suggestions! That makes sense!